app/controllers/books/comments_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Books::CommentsController < ApplicationController
+  before_action :set_book, only: %i[create destroy]
+  def create
+    if @book.comments.build(content: comment_params[:content], user: current_user).save
+      redirect_to book_url(@book), notice: t('controllers.common.notice_create', name: Comment.model_name.human)
+    else
+      render book_url(reoprt), status: :bad_request
+    end
+  end
+
+  def destroy
+    if current_user.comments.find(params[:id]).destroy
```

Comment:
1行に全部書くと「ログイン中のユーザーのコメントからidに合致するレコードを見つけてきて削除」と、一気に全部読む必要が出てきます。
個人的には「見つける」と「削除する」で、処理を分けた方が読みやすいと思います。

```suggestion
    comment = current_user.comments.find(params[:id])
    if comment.destroy
```

参考 https://twitter.com/jnchito/status/1326468865103073280

Reply:
[6ec9400](https://github.com/s-miura7/fjord-books_app-2023/pull/5/commits/6ec940087bddf0db1b89df6932afc3cae9dc99e3) で修正